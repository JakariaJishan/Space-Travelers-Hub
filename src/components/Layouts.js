import React from 'react';
import { Outlet } from 'react-router-dom';
import Nav from './Nav';

function Layouts() {
  return (
    <div>
      <Nav />
      <Outlet />
    </div>
  );
}

export default Layouts;
