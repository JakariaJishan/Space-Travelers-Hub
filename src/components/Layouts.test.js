import '@testing-library/jest-dom/extend-expect';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Layouts from './Layouts';

describe('testing Layouts', () => {
  it('all components renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Layouts />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
