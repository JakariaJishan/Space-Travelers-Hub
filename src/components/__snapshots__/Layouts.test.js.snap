// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Layouts all components renders correctly 1`] = `
<div>
  <section
    className="NavSection"
  >
    <div
      className="header"
    >
      <img
        alt="brand logo"
        src="images/planet.png"
        style={
          Object {
            "width": "10%",
          }
        }
      />
      <h1>
        Space Travelers Hub
      </h1>
    </div>
    <nav
      className="nav"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
         Rocket 
      </a>
      <a
        className=""
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
      <p>
        |
      </p>
      <a
        className=""
        href="/profiles"
        onClick={[Function]}
      >
        Profile
      </a>
    </nav>
  </section>
</div>
`;
